Projects/Contracts/Sources/Contracts.swift

```diff
@@ -107,6 +107,14 @@ extension Contracts {
                 )
             } else if case .openContractDetailErrorScreen = action {
                 ContractDetail(id: "", title: "").contractDetailErrorJourney
+            } else if case .openTerminationErrorScreen = action {
+                HostingJourney(
+                    rootView: GenericErrorView(
+                        title: "Couldn'terminate contract",
```

Comment:
Can we lokalise this?

Reply:
And do we need this since we are never calling it?

Reply:
Good catch, I'll remove it for now